Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -2559,7 +2576,273 @@ def get_indicator_dbot_object(indicator):
     return get_dbot_indicator(indicator_type, dbot, indicator_value)
 
 
-''' EXECUTION CODE '''
+''' LIVE RESPONSE CODE '''
+
+
+def run_polling_command(client: MsClient, args: dict, cmd: str, action_func: Callable,
+                        results_function: Callable, post_polling_process: Callable):
+    """
+    This function is generically handling the polling flow. In the polling flow, there is always an initial call that
+    starts the uploading to the API (referred here as the 'upload' function) and another call that retrieves the status
+    of that upload (referred here as the 'results' function).
+    The run_polling_command function runs the 'upload' function and returns a ScheduledCommand object that schedules
+    the next 'results' function, until the polling is complete.
+    Args:
+        args: the arguments required to the command being called, under cmd
+        cmd: the command to schedule by after the current command
+        upload_function: the function that initiates the uploading to the API
+        results_function: the function that retrieves the status of the previously initiated upload process
+        uploaded_item: the type of item being uploaded
+
+    Returns:
+
+    """
+    ScheduledCommand.raise_error_if_not_supported()
+    interval_in_secs = int(args.get('interval_in_seconds', 5))
```

Comment:
Minimum can't be lower than 10 (if less than 10 is provided, it'll default on 10).
```suggestion
    interval_in_secs = int(args.get('interval_in_seconds', 10))
```